pkg/util/k8s/labels.go

```diff
@@ -0,0 +1,18 @@
+package k8s
+
+import (
+	"strings"
+)
+
+// Make given string label-compatible
+func StringToLabel(s string) string {
+	// We need to remove characters that are not allowed, e.g. :
+	s = strings.Replace(s, ":", "_", -1)
```

Comment:
if  `:` is just an example,are there other chars needed to be checked/replaced, or `:` is the only relevant one?

Reply:
There are probably more of those. The usage of this is aimed on version digests being used as the label, so I focused on handling those only instead of making a generic solution fitting all. This is probably worth a doc string. Lemme add it.

Reply:
got it. Thanks